src/main/java/org/elasticsearch/index/translog/LegacyTranslogStream.java

```diff
@@ -49,6 +50,7 @@
 
     @Override
     public void write(StreamOutput out, Translog.Operation op) throws IOException {
+        // nocommit: do we want to throw an UnsupportedOperationException?
```

Comment:
I don't think we need this any more on master, since we can break backwards compatibility. In that case we can throw the UOE